Flickr/Sources/FlickrAppVCs/PhotoViewerVC.swift

```diff
@@ -107,7 +130,17 @@ class PhotoViewerVC: UIViewController {
             imageView.backgroundColor = imageViewBackgroundColor
         }
         self.imageView = imageView
-        Nuke.loadImage(with: url, into: imageView)
+        if url != nil {
+            guard let url = url else {
+                return []
+            }
+            Nuke.loadImage(with: url, into: imageView)
```

Comment:
you can use and if let and return a single time in the end of the file. Are you sure returning empty constraints for image view is a good idea?